akka-http-core/src/main/scala/akka/http/impl/engine/client/OutgoingConnectionBlueprint.scala

```diff
@@ -203,9 +219,25 @@ private[http] object OutgoingConnectionBlueprint {
         }
```

Comment:
Not sure what this comment refers to, there is no substream for `Strict` messages.